CMakeLists.txt

```diff
@@ -128,4 +129,5 @@ install( TARGETS ${PROJECT_NAME} )
 install( FILES ${CMAKE_Fortran_MODULE_DIRECTORY}/libtrixi.mod TYPE INCLUDE)
 install( FILES $<TARGET_OBJECTS:${PROJECT_NAME}_tls> TYPE LIB RENAME lib${PROJECT_NAME}_tls.o )
 install( DIRECTORY LibTrixi.jl DESTINATION share/libtrixi )
```

Comment:
`LibTrixi.jl/lib` will be installed as well. Exclude it or just ignore?

Reply:
Good point. I think ignore for now, but maybe create an issue for it to make it cleaner, eg, move to an out-of-source build?

Reply:
Fine with ignoring it.

But: Should triggering the PackageCompiler from the installed libtrixi be an option? (tests, e.g., are not installed and we only run them from the build directory)

Reply:
No, I’d say that it’s either PackageCompiler or our C library. I would treat them as mutually exclusive, not a composition. So you either use our C library or the PC approach during installation. For that, we also need a new CMakeLists.txt (or better yet, a selector whether you want to use the C shim or PC) and then fix the installation part accordingly. What’s also missing right now is the ability to use the Fortran examples.

maybe you can take a stab at integrating the PC option into our CMake setup? 😬

Reply:
Sounds like a nice idea. Moved to #124.